docs/error_pages/404.rst

```diff
@@ -0,0 +1,8 @@
+ Error 404
+===========
+
+The page you requested does not exist.
+
+The page may have moved.
+
+Try searching or click the navigation links on the left.
```

Comment:
Navigation links aren't always on the left (on small window sizes they are above, in a menu).